content/docs/introduction/faq.mdx

```diff
@@ -188,7 +199,7 @@ Everything in the `tina` directory (except the `__generated__` folder) needs to
 
 ### How do I resolve `API URL is misformatted` errors?
 
-The ContentAPI URL isn't formatted correctly. See [here](/docs/tina-cloud/overview/#enabling-tina-cloud-in-tinacms) for information on how the URL *should* be formatted.
+The ContentAPI URL isn't formatted correctly. See [here](/docs/tina-cloud/overview/#enabling-tina-cloud-in-tinacms) for information on how the URL _should_ be formatted.
```

Comment:
Hi @bradystroud , 

- [ ] Can you change from

> See [here](/docs/tina-cloud/overview/#enabling-tina-cloud-in-tinacms) for information on how the URL _should_ be formatted.

To

> See information on [how the URL should be formatted](/docs/tina-cloud/overview/#enabling-tina-cloud-in-tinacms).

---

As per https://www.ssw.com.au/rules/descriptive-links